lib/MiLightState/GroupState.h

```diff
@@ -115,6 +115,7 @@ class GroupState {
         _isSetState           : 1,
         _isSetHue             : 1;
       uint32_t
+        _isNightMode          : 1,
```

Comment:
Why are we moving this?  I think any changes to the ordering of the fields here will break backwards compatibility.